prometheuspvesd/test/unit/test_discovery.py

```diff
@@ -97,8 +87,8 @@ def test_validate_ip(discovery, addresses):
         assert not discovery._validate_ip(address)
 
 
-def test_get_ip_addresses(mocker, discovery, networks):
-    discovery.client.get.side_effect = lambda *args: get_mock(networks, *args)
+def test_get_ip_addresses(discovery, networks):
+    discovery.client.get_network_interfaces = MagicMock(return_value=networks)
```

Comment:
Can we use the mocker instead?

`mocker.patch.object(ProxmoxClient, "get_network_interfaces", return_value=networks)`